src/lib/utils/gh.ts

```diff
@@ -1,28 +1,33 @@
+import fs from "fs"
 import { join } from "path"
+import { execSync } from "child_process"
 
-import { LAST_COMMIT_BASE_URL, OLD_CONTENT_DIR } from "../constants"
+import { CONTENT_DIR, DEFAULT_LOCALE, TRANSLATIONS_DIR } from "../constants"
 
-export const getLastModifiedDate = async (filePath: string, locale: string) => {
-  const headers = new Headers({
-    // About personal access tokens https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#about-personal-access-tokens
-    Authorization: "Token " + process.env.GITHUB_TOKEN_READ_ONLY,
```

Comment:
Should we remove this env var now? maybe we can leave it if we feel that we will need it for other things